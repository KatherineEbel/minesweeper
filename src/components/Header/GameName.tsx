import React from 'react'
import tw from 'twin.macro'

export interface GameNameProps {
  text: string
}

const StyledH1 = tw.h1`text-3xl font-display tracking-wider mb-4`

const GameName = ({ text }: GameNameProps) => {
  return <StyledH1>{text}</StyledH1>
}

export default GameName
