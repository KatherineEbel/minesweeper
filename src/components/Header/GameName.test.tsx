import React from 'react'
import { render } from '@testing-library/react'

import GameName from '@/components/Header/GameName'

test('GameName renders correctly', () => {
  const { asFragment } = render(<GameName text='minesweeper'/>)
  expect(asFragment()).toMatchSnapshot()
})