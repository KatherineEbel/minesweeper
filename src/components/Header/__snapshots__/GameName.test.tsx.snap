// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GameName renders correctly 1`] = `
<DocumentFragment>
  <h1
    class="css-9a8zuw"
  >
    minesweeper
  </h1>
</DocumentFragment>
`;
